main.go

```diff
@@ -20,27 +19,53 @@ import (
 )
 
 type SlackStatus struct {
-	StatusText  string `json:"status_text"`
-	StatusEmoji string `json:"status_emoji"`
+	StatusText  string `mapstructure:"status_text" json:"status_text"`
```

Comment:
Adding the `mapstructure` struct tags to all of the configuration struct fields makes marshalling more reliable.